src/Features/AnalyticsDashboard.php

```diff
@@ -88,6 +88,7 @@ public function register_page() {
 				'parent'       => 'woocommerce',
 				'path'         => self::MENU_SLUG,
 				'is_top_level' => true,
+				'order'        => 1,
```

Comment:
Does zero work here? I'd hate for someone to sneak into the top spot.

Reply:
It does!  Updated to 0.